src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -243,7 +250,11 @@
   [server :- Server
    config :- config/WebserverConnector]
   (let [request-size (:request-header-max-size config)]
-    (doto (ServerConnector. server (connection-factory request-size))
+    (doto (ServerConnector. server
+                            nil nil nil
```

Comment:
Ah, I just realized I was looking at the 9.2.6 docs. It was probably added sometime between the version we're using and 9.2.6.